DotNet/emitter.js

```diff
@@ -136,16 +138,24 @@
 	if ("parentBranchName" in test.config) addHook('beforeEach', dot_net`eyes.ParentBranchName = ${test.config.parentBranchName};`)
 	if ("hideScrollbars" in test.config) addHook('beforeEach', dot_net`eyes.HideScrollbars = ${test.config.hideScrollbars};`)
 	if ("isDisabled" in test.config) addHook('beforeEach', dot_net`eyes.IsDisabled = ${test.config.isDisabled};`)
+	if ("batch" in test.config) {
+		if ("id" in test.config.batch) {
+			addHook('beforeEach', dot_net`eyes.Batch.Id = ${test.config.batch.id};`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
			addHook("beforeEach", dot_net`eyes.Batch.Id = ${test.config.batch.id};`)
```